README.md

```
@@ -1,219 +1,256 @@
+✅ **Audited**
+✅ **Observable**
+✅ **Callable via MCP**
 
-> This will prompt you to log in or to create an account on
-> [decocms.com](https://decocms.com).
+---
 
-2. Enter the project directory and start the dev server
+## 🧱 Core Architecture
 
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)